airsonic-main/src/main/java/org/airsonic/player/service/UPnPService.java

```diff
@@ -122,7 +122,7 @@ private void startService() {
     }
 
     private synchronized void createService() {
-        upnpService = new UpnpServiceImpl(new DefaultUpnpServiceConfiguration(SettingsService.getDefaultUPnPPort()));
+        upnpService = new UpnpServiceImpl(new ApacheUpnpServiceConfiguration(SettingsService.getDefaultUPnPPort()));
```

Comment:
I do not see much of a difference between this custom class and the parent class except for an request executor. I'm not convinced we know the issue.

Reply:
Scratch that, I think I see what you're saying: the packages are different for the similarly named the classes